src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.js

```diff
@@ -3,6 +3,7 @@ import { InjectAppServices } from '../../../services/pure-di';
 import { FormattedMessage, FormattedNumber } from 'react-intl';
 import Loading from '../../Loading/Loading';
 import * as S from './ReportsTrafficSources.styles';
+import { BoxMessage } from '../../styles/messages';
```

Comment:
import * as S from '../../styles/messages';

Reply:
There is already an `S`:

![image](https://user-images.githubusercontent.com/1157864/65323637-4ca9bd00-db80-11e9-9035-1571471e38ba.png)

We have talked with @GusBaamonde that in the long term we need more reusable styles, for example, there should not be `ReportsTrafficSources.styles`, I mean styles specific for traffic sources.